server/cli/src/refresh_dates.rs

```diff
@@ -83,6 +83,7 @@ fn get_exclude_timestamp_fields() -> Vec<TableAndFieldName> {
 fn get_date_fields() -> Vec<TableAndFieldName> {
     vec![
         ("name", "date_of_birth"),
+        ("name", "date_of_death"),
```

Comment:
Is this the query method for getting data from mSupply?

Reply:
No, it's for refreshing dates to keep data current...

Reply:
Yes this if for a command line interface feature to take an existing "demo" database and move the date forward to get an up-to-date dataset. Also have a look at the RefreshDates option in `cli/src/cli.rs`.